apps/condo/domains/ticket/hooks/useTicketDocumentGenerationTask.tsx

```diff
@@ -45,15 +46,22 @@ export const useTicketDocumentGenerationTask = ({ ticket, user, timeZone }) => {
         handleRunTask({ taskAttrs: { documentType } })
     }, [handleRunTask, loading])
 
+    const isValidInvoice = invoices.find((invoice) => get(invoice, 'status') !== 'Canceled' &&  get(invoice, 'deletedAt') === null)
```

Comment:
use const from `domains/marketplace/constants`:
```suggestion
    const isValidInvoice = invoices.find((invoice) => get(invoice, 'status') !== INVOICE_STATUS_CANCELED &&  get(invoice, 'deletedAt') === null)
```